GDJS/Runtime/pixi-renderers/layer-pixi-renderer.js

```diff
@@ -250,3 +255,33 @@ gdjs.LayerPixiRenderer.prototype.isEffectEnabled = function(name) {
 
   return gdjs.PixiFiltersTools.isEffectEnabled(filter);
 };
+
+gdjs.LayerPixiRenderer.prototype.updateRenderTexture = function() {
+  if(!this._renderTexture) {
+    var width = this._pixiRenderer.width;
+    var height = this._pixiRenderer.height;
+    var resolution = this._pixiRenderer.resolution;
+    this._renderTexture = PIXI.RenderTexture.create({
+      width,
+      height,
+      resolution
+    });
+  }
+
+  if(this._oldWidth !== this._pixiRenderer.width || 
+    this._oldHeight !== this._pixiRenderer.height) {
+      this._renderTexture.resize(
+        this._pixiRenderer.width,
+        this._pixiRenderer.height
+      )
+      this._oldWidth = this._pixiRenderer.width;
+      this._oldHeight = this._pixiRenderer.height;
+  }
+
+  this._pixiRenderer.render(this._pixiContainer, this._renderTexture, false);
+}
```

Comment:
@4ian what do you think about this? I guess maybe having a texture this big is the reason for slow fillrate? But rendering a layer to texture would mean rendering the entire screen, right? Either way, I'll maybe checkout a new branch from this point (considering you don't feel like something's wrong here) to try out other things (like actually using this texture in a test game).